### PR TITLE
PMM-3649 Rhel8 changes

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-191-e62ea34
client docker - perconalab/pmm-client-fb:PR-191-e62ea34
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm2-client/pmm2-client-PR-191-e62ea34.tar.gz

File: build/bin/build-client-packages [hunk 1]
```diff
@@ -107,7 +107,24 @@ check_workdir(){
 add_percona_yum_repo(){
     if [ ! -f /etc/yum.repos.d/percona-dev.repo ]
     then
-        cat >/etc/yum.repos.d/percona-dev.repo <<EOL
+        if [ $RHEL = 8 ]; then
```
Comment (hors) [inline build/bin/build-client-packages]: "x$RHEL" = "x8"

Comment (EvgeniyPatlan) [reply build/bin/build-client-packages]: fixed

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-191-c307849
client docker - perconalab/pmm-client-fb:PR-191-c307849
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm2-client/pmm2-client-PR-191-c307849.tar.gz

**Pull request merged**